talisker/endpoints.py

```diff
@@ -143,10 +143,13 @@ def __init__(self, app, namespace='_status'):
         if pkg_is_installed('prometheus-client'):
             self.urlmap['/metrics'] = 'metrics'
             self.urlmap['/test/prometheus'] = 'test_prometheus'
+        # Publish /info/logtree only if logtree is available
         if pkg_is_installed('logging-tree'):
             self.urlmap['/info/logtree'] = 'logtree'
+        # Publish /info/workers only if psutil is available
         if pkg_is_installed('psutil'):
             self.urlmap['/info/workers'] = 'workers'
+        # Publish /info/objgraph only if objgraph is available
```

Comment:
I don't think these comments are necessary - they say the same things as: pkg_is_install('xxx'), IMO

Reply:
no problem. Will remove these comments when I'm gonna look at travis-ci errors.